samples/grids/grid/clipboard-operations/src/index.ts

```diff
@@ -42,9 +43,10 @@ export class Sample {
                 grid.cellSelection = 'multiple';
             });
 
+            this.defaultSeparator = grid.clipboardOptions.separator;
```

Comment:
Sample throws error:
![image](https://github.com/user-attachments/assets/bc3d849d-03c9-453a-a3d5-d268bc3f3cae)

Additionally, the igcChange event seems to not trigger for some reason, so the default separator is used regardless of what is in entered into the input.

Same issue applies to the tree grid clipboard operations sample

Reply:
Should be ok now